app/src/test/scala/org/alephium/explorer/ExplorerSpec.scala

```diff
@@ -293,6 +293,25 @@ trait ExplorerSpec
     }
   }
 
+  "get all transactions for addresses" in {
+    forAll(Gen.someOf(transactions)) { transactions =>
+      val limitedAddresses = transactions.flatMap(_.outputs.map(_.address)).take(txLimit)
+      val addressesBody    = limitedAddresses.map(address => s""""$address"""").mkString("[", ",", "]")
```

Comment:
it's a shame, but I actually discovered yesterday that you can pass 3 args to `mkString` and not only 1 :sweat_smile: